serverless.yml

```diff
@@ -125,6 +125,9 @@ functions:
             parameters:
               paths:
                 hash: true
+          caching:
```

Comment:
This won't enable caching on this endpoint unless we add `enabled: true`, right?

I'm not sure if you wanted to enabled caching, but I think we shouldn't enable in endpoints that didn't have it enabled before, unless you have a good reason for this.

If you didn't want to enable caching, but only make it future-proof (in case someone enables caching in the future), then I think we should just leave a comment to make it explicit that the current configuration doesn't enable caching, because someone reading the code will assume caching is enabled by this (at least until further looking into the plugin docs)

Does it make sense?

Reply:
You're right. We might not put caching properties on APIs that do not have cache enabled. I am sending a new version correcting this part.